app/embeds/GoogleDataStudio.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+import * as React from "react";
+import Image from "components/Image";
+import Frame from "./components/Frame";
+
+const URL_REGEX = new RegExp(
+  "^https?://datastudio.google.com/embed/reporting/(.*)/page/(.*)$"
+);
+const URL_REGEX_SECONDARY = new RegExp(
+  "^https?://datastudio.google.com/u/0/reporting/(.*)/page/(.*)/edit?$"
+);
+
+type Props = {|
+  attrs: {|
+    href: string,
+    matches: string[],
+  |},
+|};
+
+export default class GoogleDataStudio extends React.Component<Props> {
+  static ENABLED = [URL_REGEX] || [URL_REGEX_SECONDARY];
+
+  render() {
+    return (
+      <Frame
+        {...this.props}
+        src={
+          URL_REGEX_SECONDARY
```

Comment:
Similar, this logic doesn't make sense – the regex will always evaluate to truthy here and the second case is never ran